### ISSUE TITLE
About parameter on Pascal.

### ISSUE THREAD

**Issue by prettydong** [snucml/LargeLossMatters#3]

**Comment by prettydong** [comment]
Your work is very interesting !
would you share precision parameters on Pascal with me ? Thank you!

**Comment by prettydong** [comment]
I use rel 0.2, lr 1e-5, backbone imgaenetV2 of resnet50 , epoch 15.
then i get Test mAP : 89.124 😊

**Comment by jameslahm** [comment]
Hi, It's under "End-to-End" training setting?

**Comment by prettydong** [comment]
> Hi, It's under "End-to-End" training setting?

Yes, 'end2end' is defualt setting. 
I run the code on a PC with single RTX3060 12G not 4GPUs in paper. 
And it seems that learning rate of last linear layer is different from resnet backbone.

**Comment by jameslahm** [comment]
It seems that the paper also uses a single GPU on pascal too and only use 4 GPUs on openimages. 
> And it seems that learning rate of last linear layer is different from resnet backbone.

Yes, the learning rate of last linear layer is 10 times larger than resnet backbone.

**Comment by jameslahm** [comment]
> I use rel 0.2, lr 1e-5, backbone imgaenetV2 of resnet50 , epoch 15.
> then i get Test mAP : 89.124 😊

Hi, @prettydong , did you use `LL-R` method？I try these hyperparameters under LL-R and only get 87.795 test mAP. Did you change somewhere else? Thanks!

**Comment by prettydong** [comment]
> > I use rel 0.2, lr 1e-5, backbone imgaenetV2 of resnet50 , epoch 15.
> > then i get Test mAP : 89.124 😊
> 
> Hi, @prettydong , did you use `LL-R` method？I try these hyperparameters under LL-R and only get 87.795 test mAP. Did you change somewhere else? Thanks!

I use `LL-R` under 0.2rel and 1e-5 lr ,other parameter is defualt. Then I can get 88 mAP. The torch warning me that I use old backbone , then I use `IMAGENET1K_V2` which resize image to 232, you can find details on "https://github.com/pytorch/vision/issues/3995#issuecomment-1013906621". 
->`models.py:22`---`feature_extractor = torchvision.models.resnet50(weights=ResNet50_Weights.DEFAULT)`.
This backbone spends more times. And best performance occures on 12-13th epoch. Then mAP get 89.1.
I plan to do some incremental work on this paper. 😂I'd be happy to discuss the details of this paper with you.

**Comment by youngwk** [comment]
Thanks for your interest.
For LL-R, I used delta_rel=0.1, lr=2.5E-03 with SGD optimizer.

**Comment by prettydong** [comment]
Thanks！

**Comment by jameslahm** [comment]
> > > I use rel 0.2, lr 1e-5, backbone imgaenetV2 of resnet50 , epoch 15.
> > > then i get Test mAP : 89.124 😊
> > 
> > 
> > Hi, @prettydong , did you use `LL-R` method？I try these hyperparameters under LL-R and only get 87.795 test mAP. Did you change somewhere else? Thanks!
> 
> I use `LL-R` under 0.2rel and 1e-5 lr ,other parameter is defualt. Then I can get 88 mAP. The torch warning me that I use old backbone , then I use `IMAGENET1K_V2` which resize image to 232, you can find details on "[pytorch/vision#3995 (comment)](https://github.com/pytorch/vision/issues/3995#issuecomment-1013906621)". ->`models.py:22`---`feature_extractor = torchvision.models.resnet50(weights=ResNet50_Weights.DEFAULT)`. This backbone spends more times. And best performance occures on 12-13th epoch. Then mAP get 89.1. I plan to do some incremental work on this paper. 😂I'd be happy to discuss the details of this paper with you.

Thanks! I'd be happy too!
